File: C++/M01_ProgrammingBasics/L02_ConditionalStatements/Lab/Solutions/P06_AreaOfFigures.cpp
#include <iostream>
using namespace std;

int main() {

	string shape;
	cin >> shape;

	double area = 0;
	if (shape == "square") {
		double side;
		cin >> side;
		area = side * side;
	} else if (shape == "rectangle") {
		double side1, side2;
		cin >> side1 >> side2;
		area = side1 * side2;
	} else if (shape == "circle") {
		double pi = 3.14159265359;
		double r;
		cin >> r;
		area = r * r * pi;
	} else if (shape == "triangle") {
		double side, height;
		cin >> side >> height;
		area = side * height / 2;
	}

	cout.setf(ios::fixed);
	cout.precision(3);

	cout << area << endl;

	return 0;
}

File: Python/M01_ProgrammingBasics/L02_ConditionalStatements/Lab/Solutions/P06_AreaOfFigures.py
import math

item = input()

square = 0
rectangle = 0
circle = 0
triangle = 0

if item == 'square':
    square_item = float(input())
    size = square_item ** 2
    print(f'{size:.3f}')
if item == 'rectangle':
    rectangle_item1 = float(input())
    rectangle_item2 = float(input())
    size = rectangle_item1 * rectangle_item2
    print(f'{size:.3f}')
if item == 'circle':
    circle_item = float(input())
    size = math.pi * (circle_item * circle_item)
    print(f'{size:.3f}')
if item == 'triangle':
    triangle_item1 = float(input())
    triangle_item2 = float(input())
    size = triangle_item1 * (triangle_item2 / 2)
    print(f'{size:.3f}')